Project/CMake/CMakeLists.txt

```diff
@@ -423,6 +423,10 @@ else()
     SOVERSION ${MediaInfoLib_SOVERSION})
 endif()
 
+set_property(TARGET mediainfo PROPERTY CXX_STANDARD 11)

```

Comment:
What is the difference between this line and just `CXX_STANDARD 11` (what you have done in ZenLib corresponding PR)?

Reply:
i wasn't sure is possible to call set_target_properties twice.

Fixed